### PR TITLE
DB-10999 fix MAXIMUMDISPLAYWIDTH / refactor more ij.jj

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: sqlshell.sh [hunk 1]
```diff
@@ -9,14 +9,25 @@ else
...
+    fi
+    export MAVEN_OPTS="${MAVEN_OPTS} -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=$port"
+fi
+
+
 if [ -z "${CLIENT_SSL_KEYSTORE}" ]; then
   cd splice_machine ; ${RLWRAP} mvn exec:java ; cd ${DIR}
 else
-  cd splice_machine ; ${RLWRAP} mvn  exec:java \
+  cd splice_machine ; ${RLWRAP} MAVEN_OPTS="${MY_MAVEN_OPTS}" mvn exec:java \
```
Comment (msirek) [inline sqlshell.sh:30]: Is there a reason to only support the new debug command-line option when we are using a CLIENT_SSL_KEYSTORE?

Comment (dgomezferro) [reply sqlshell.sh:30]: I think this is an error, `MY_MAVEN_OPTS` is not defined

**Review by dgomezferro — CHANGES_REQUESTED** [review]
Nice cleanup! I think there's an error using `MY_MAVEN_OPTS`, looks good otherwise

**Comment by martinrupp** [comment]
> Nice cleanup! I think there's an error using `MY_MAVEN_OPTS`, looks good otherwise

yes, you're correct, i fixed it. also now works for both SSL and non-SSL

**Review by dgomezferro — APPROVED** [review]

**Review by msirek — APPROVED** [review]
+1

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**